### ISSUE TITLE
Guide for migrating from AdamW

### ISSUE THREAD

**Issue by 51616** [zhuhanqing/APOLLO#9]
Thank you for such a cool project.

If I want to migrate from AdamW (or other optimizers), what is the rule of thumb of adjusting the hyperparameters (e.g, learning rate, weight decay, etc.)?
